### PR TITLE
Logo generation via python

### PR DESCRIPTION
**Pull request by MattIPv4**
Generates all our logo variants through a Python script from source SVGs and a template README file.

### CODE REVIEW

**Comment by MattIPv4** [comment]
@PeterDaveHello Could you approve and merge this, please? :)

File: logo/source/generation.py [hunk 1]
```diff
@@ -0,0 +1,354 @@
+"""
```
Comment (PeterDaveHello) [inline logo/source/generation.py]: should add a shebang here like: `#!/usr/bin/env python3`?

**Comment by MattIPv4** [comment]
TODO: Needs a compression script implemented as part of the generation process.

**Comment by MattIPv4** [comment]
@PeterDaveHello This now has a clean commit history and makes use of PIL's compression to ensure the files are as small as they can be. I ended up not using zopfli as no one seems to have decent bindings for it in Python, the only library that does fails to install. As this is the main repo for all assets, I feel that high levels of compression aren't needed and can be applied where the assets are used if needed.

File: logo/source/generation.py [hunk 2]
```diff
@@ -269,7 +269,7 @@ def __save_all(logos: List[Logo], directory: str, *args, **kwargs):
             filename = logo.save(directory, *args, **kwargs)
             items.append([logo, filename])
 
-        items = ["|{}|{}|{}|{}|".format(
+        items = ["| {} | {} | {} | {} |".format(
```
Comment (PeterDaveHello) [inline logo/source/generation.py]: Maybe this is part of previous commit?

Comment (MattIPv4) [reply logo/source/generation.py]: Fixed @PeterDaveHello :)

**Comment by PeterDaveHello** [comment]
@MattIPv4 can you please tell me how to verify the image generating process? I got syntax error like this:
```
$ python3 logo/source/generation.py 
  File "logo/source/generation.py", line 55
    dark: Color = Color(69, 70, 71, 255)  # 454647
        ^
SyntaxError: invalid syntax
```

**Comment by MattIPv4** [comment]
@PeterDaveHello This runs on python 3.6 or 3.7 (needs support for type hinting). Running the file should then just generate everything.

**Comment by PeterDaveHello** [comment]
@MattIPv4 should we add a `requirements.txt` about the python dependencies being used?

**Comment by MattIPv4** [comment]
@PeterDaveHello The requirements.txt file is present alongside the generation script.

**Comment by PeterDaveHello** [comment]
Oops, I missed it.

**Pull request merged**